os-client/src/main/kotlin/io/provenance/os/client/OsClient.kt

```diff
@@ -86,18 +86,6 @@ open class OsClient(
             }
     }
 
-    fun get(uri: String, publicKey: PublicKey): DIMEInputStream {
```

Comment:
was this just no longer valid with the new object store or something?

Reply:
🤷  I noticed nothing in p8e is using it right now. I'm not sure if it was used in the past or not.